test/schemaValidation.test.ts

```diff
@@ -96,7 +96,9 @@ suite('Validation Tests', () => {
                 }).then(done, done);
             });
 
-            it('Null scalar value should be treated as string', done => {
+            // TODO: remove this test, because it is just plain wrong: https://yaml.org/type/null.html
+            // See: https://github.com/redhat-developer/yaml-language-server/issues/118
+            it.skip('Null scalar value should be treated as string', done => {
```

Comment:
I think instead of skipping we should probably just removed it at this point since that's technically a violation of the spec

Reply:
Yes! I just left it there to not be too destructive, wanted someone to double check. I will remove it.